tests/base/src/test/java/org/wildfly/security/auth/realm/token/JwtSecurityRealmTest.java

```diff
@@ -845,4 +801,21 @@ public MockResponse dispatch(RecordedRequest recordedRequest) {
             }
         };
     }
+
+    private void testTokenSecurityRealm(String kid, int port) throws Exception {
```

Comment:
Would be good to rename this to explain what the method does, e.g., checkIdentityDoesNotExist or testIdentityDoesNotExist, etc.